Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers.yml

```diff
@@ -1343,37 +1343,37 @@ inputs:
     simple: "2"
   required: true
   description: |-
-    Set the minimum severity of an incident with a BCL (bulk sender) that has a score equal to or higher than 4.
+    Sets the minimum severity of an incident with a Bulk Complaint Level (BCL) score equal to or greater than 4.
 
-    Available values: 0 (Unknown), 1 (Low), 2 (Medium), 3 (High), 4 (Critical).
+    Possible values: 0 (Unknown), 1 (Low), 2 (Medium), 3 (High), 4 (Critical).
   playbookInputQuery:
 - key: PCL-Severity
   value:
     simple: "2"
```

Comment:
This is the intended default value?

Reply:
Yes - this is the minimum severity set for an incident when the score is higher then 4 (it is written in the description - we went through this when I was building this playbook).